lecture/models.py

```
@@ -44,26 +44,27 @@ class LecturePDF(BaseModel):
         related_name='pdf_files'
     )
     pdf_file = models.FileField(upload_to='lecture_pdfs/')
-    pdf_title= models.TextField(null=True, blank=True)
+    pdf_title = models.TextField(null=True, blank=True)
 
     def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)